NOTICES.txt

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
There's already a NOTICE.txt, we should merge them

Reply:
Good catch!